core/src/main/java/tc/oc/pgm/rotation/MapPoll.java

```diff
@@ -136,7 +136,7 @@ private Component getMapChatComponent(MatchPlayer viewer, MapInfo map, boolean w
         .build();
   }
 
-  public void sendBook(MatchPlayer viewer) {
+  public void sendBook(MatchPlayer viewer, boolean force) {
```

Comment:
```suggestion
  public void sendBook(MatchPlayer viewer, boolean forceOpen) {
```
force in this context could mean if the book is forced, i think making it more explicit can help